package.json

```diff
@@ -33,29 +33,30 @@
   "prettier": "@guardian/prettier",
   "dependencies": {
     "@creditkarma/thrift-server-core": "^0.16.1",
-    "@emotion/babel-preset-css-prop": "^10.1.0",
-    "@emotion/core": "^10.0.35",
-    "@emotion/styled": "^10.0.27",
+    "@emotion/cache": "^11.1.3",
+    "@emotion/css": "^11.1.3",
+    "@emotion/react": "^11.1.5",
+    "@emotion/server": "^11.0.0",
     "@guardian/apps-rendering-api-models": "^0.11.0",
-    "@guardian/atoms-rendering": "^9.0.0",
+    "@guardian/atoms-rendering": "^12.2.0",
     "@guardian/bridget": "^1.8.0",
     "@guardian/content-api-models": "^15.10.0",
     "@guardian/content-atom-model": "^3.2.4",
     "@guardian/discussion-rendering": "^4.6.1",
-    "@guardian/image-rendering": "github:guardian/image-rendering#6ad32f87",
+    "@guardian/image-rendering": "github:guardian/image-rendering#1167f8bc",
```

Comment:
Should we have merged this in rather than depending on a branch?

Reply:
We could, was planning to do that next or might even just not release image-rendering's `v7` since v6 already has bumped up types that we need to do soon after this.